.fcrepo_wrapper

```diff
@@ -0,0 +1,10 @@
+# Place any default configuration for fcrepo_wrapper here
+port: 8984
+enable_jms: false
+
+download_dir: tmp/fcrepo_wrapper
```

Comment:
Given the order of the files here, and the use of `Hash#merge!`, I believe app-specific config will always override `~` config: https://github.com/cbeer/fcrepo_wrapper/blob/master/lib/fcrepo_wrapper/configuration.rb#L121-L125